resources/weapons/bombs/Mk-83.yaml

```diff
@@ -1,16 +1,17 @@
-name: Mk 83
-clsids:
-  - "{MAK79_MK83 1R}"
-  - "{MAK79_MK83 1L}"
-  - "{BRU-32 MK-83}"
-  - "{Mk_83BT}"
-  - "{Mk_83CT}"
-  - "{Mk_83P}"
-  - "{Mk_83PT}"
-  - "{BRU42_MK83 RS}"
-  - "{BRU3242_MK83 RS}"
-  - "{PHXBRU3242_MK83 RS}"
-  - "{7A44FF09-527C-4B7E-B42B-3F111CFE50FB}"
-  - "{BRU42_MK83 LS}"
-  - "{BRU3242_MK83 LS}"
-  - "{PHXBRU3242_MK83 LS}"
+name: Mk 83

```

Comment:
It looks like your git client isn't configured with https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#global-settings-for-line-endings. I just added a `.gitattributes` file to enforce it. After rebasing your changes onto develop you'll need to run `git add --renormalize .` to fix these.

(why does this matter? it makes the diff harder to read if every line in the file changes. afaict the thing that changed here is that a fallback was added, but it's hard to tell)

Reply:
You are correct - I added a fallback to this and the Mk-84. I didn't use a standalone client. I may have worked all of this incorrectly. I created a fork in Github, created or edited yaml files, uploaded them to the fork, then issued a pull request. I'll spend some time figuring out git and, going forward, will work any further changes through an editor

Reply:
If you don't figure it out I can fix it for you before you submit easily enough. After you update your branch with he new changes in develop it'll be automatic on your end, but only for the files you touch from then on.